src/reference-assemblies/Directory.Build.props

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <DisableBuildToolsRoslynVersion>true</DisableBuildToolsRoslynVersion>
```

Comment:
Is this related to the tarball or just a general build fix?

Reply:
It applies to reference-assemblies, which are only built in the Offline build.

Reply:
Is it causing a particular problem while building the ref-assemblies? If so can you give a note in a comment here of why it is disabled.